content/lectures/final_project.md

```diff
@@ -0,0 +1,39 @@
+Title: Final Project Instructions
+Summary:  Final Project
+Date: 10/31/2017
+Tags: final, project
+Category: lectures
+Author: Kerry Key and Ryan Abernathey
+
+
+
+** Final Project **
+
+
+The final project is intended to be an independent project that uses the skills you have learned in this class, as well as things you have learned on your own, to create a complicated and interesting data analysis program that accomplishes one or more research-relevant tasks. Unlike the homework assignments where we supplied the data files and told you what to do, now it is your turn to identify a  data set relevant to your own research interests and to create a code that will analyze that data and present  results in graphical form.
```

Comment:
Rather than "project that uses..."

What about "project that demonstrates master of..."

Reply:
mastery might be too strong given they just learned this stuff, but definitely need to have learned the commands!